src/main/java/io/suhan/lotto/model/lotto/LottoFactory.java

```diff
@@ -0,0 +1,26 @@
+package io.suhan.lotto.model.lotto;
+
+import io.suhan.lotto.model.NumberPool;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class LottoFactory {
+    public static Lotto createLotto(NumberPool pool) {
```

Comment:
```suggestion
    private static Lotto createLotto(NumberPool pool) {
```

+ 메서드 순서도 변경해야할 것 같아요~

Reply:
확인 및 수정 완료했습니다!

https://github.com/next-step/java-lotto-clean-playground/pull/141/commits/e4c6574db005d79d0ceb385adebb8649d57c249e 커밋에 반영 완료